<launch>
  <arg
    name="model" 
    default="$(find rover19_urdf)/urdf/rover19.urdf" />
  <arg
    name="gui"
    default="False" />
  <arg name="rvizconfig" 
       default="$(find urdf_tutorial)/rviz/urdf.rviz" />
  <param
    name="robot_description"
    textfile="$(find rover19_urdf)/urdf/rover19.urdf" />
  <param
  
    name="use_gui"
    value="$(arg gui)" />
  <node
    name="joint_state_publisher"
    pkg="joint_state_publisher"
    type="joint_state_publisher">
    <param name="/use_gui" value="true"/>
    <rosparam param="/source_list">[/URDF_JSP_pub]</rosparam> 
  </node>
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="state_publisher" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find rover19_urdf)/urdf.rviz" 
    required="true" />
  <node name="JSP"
      pkg="rover19_urdf"
      type="JSP" 
      output="screen" 
      required="true" />
  <node name="Odometery_new"
      pkg="rover19_urdf"
      type="Odometry_new" 
      output="screen" 
      required="true" />
  <node name="arm_angs"
      pkg="rover19_urdf"
      type="arm_angs.py" 
      output="screen" 
      required="true" />
</launch>